ipatests/test_webui/test_group.py

```diff
@@ -1,3 +1,4 @@
+#testing
```

Comment:
What is the purpose of this line?

Reply:
Sorry, By Mistake I have added. Now I have removed